src/pcm.cpp

```diff
@@ -79,7 +79,7 @@ double getAverageUncoreFrequencyGhz(const UncoreStateType& before, const UncoreS
     return getAverageUncoreFrequency(before, after) / 1e9;
 }
 
-void print_help(const string prog_name)
+void print_help(const string & progname)
 {
     cerr << "\n Usage: \n " << progname
```

Comment:
How the heck did this ever compile? Is there a global progname variable or so?

Reply:
same here. Please keep the old name.